### DISCUSSION TITLE
Can't install brew.

### DISCUSSION THREAD

**Discussion by xmegadezerx** [Homebrew/discussions#3080] [Getting started] [answered]
Trying to install brew on my virtual machine CentOS9.
At the end of installation:

Error: Failed to install ruby 2.6.8!
Error: Failed to install Homebrew Portable Ruby and cannot find another Ruby 2.6.8!

**Comment by gromgit** [comment, marked as answer]
> ```
> Error: Failed to install ruby 2.6.8!
> Error: Failed to install Homebrew Portable Ruby and cannot find another Ruby 2.6.8!
> ```

Please post the _full_ output of your install.

**Reply by xmegadezerx** [reply]
Checked again - somehow I missed that there was a line that tar command can't be found.
Installed tar and all went well. My bad.
